packages/frontend/src/components/ui/OfferItem/OfferItem.tsx

```diff
@@ -0,0 +1,124 @@
+import { Chip } from '@/components/ui/Chip';
+import { Button } from '@/components/ui/Button';
+
+export interface IOfferItem {
+  type: string;
+  status?: string;
+  price: number;
+  createdAt: string;
+  createdBy: {
+    userId: string;
+    username: string;
+  };
+  tradeOffer: {
+    comicId: string;
+    title: string;
+  };
+}
+
+interface IOfferItemProps extends IOfferItem {
+  index: number;
+}
+
+export const OfferItem = ({
+  index,
+  status,
+  type,
+  price,
+  createdAt,
+  createdBy,
+  tradeOffer,
+}: IOfferItemProps) => {
+  const DUMMY_COMIC_DETAILS = {
+    issue: 8,
+    imageUrl:
+      'https://cdn.marvel.com/u/prod/marvel/i/mg/3/60/6227804f029af/clean.jpg',
+  };
+
+  if (status === 'fulfilled') {
+    return <> </>;
```

Comment:
if you don't want to return anything, return `null`

Reply:
done, thanks @fmtabbara